cypress/integration/LoginForm.js

```diff
@@ -5,11 +5,11 @@ import {
     checkPermanentAddressInResult,
     fillInTheRegistrationFormAndClickTheSendButton
 } from "../models/loginPage";
-import {BASE_URL, FAKE_DATA} from "../testData/testData";
+import {BASE_URL, FAKE_DATA, LOGIN_PAGE_ENDPOINT} from "../testData/testData";
 
 describe("Verify that the login form works correctly", () => {
     beforeEach(() => {
-        cy.visit(BASE_URL);
+        cy.visit(BASE_URL + LOGIN_PAGE_ENDPOINT);
```

Comment:
pls add base url into cypress.json file. 
u can use it in tests as ` cy.visit('/'+ LOGIN_PAGE_ENDPOINT)`

Reply:
Done